scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -58,6 +58,18 @@ cargo-fmt:
     - cargo +nightly fmt --all -- --check
     - rusty-cachier cache upload
 
+cargo-fmt-toml:
+  stage: test
+  extends:
+    - .docker-env
+    - .test-refs
+  script:
+    - cargo install zepter --locked --version 0.9.2 -q -f --no-default-features && zepter --version
+    - zepter format features --workspace --fix
```

Comment:
The tool should be documented in the contribution guides or similar and how to use it

Reply:
Better to add a bash script and run it in ci for local dev?

Reply:
WDYM? Running `zepter format features` (the `--workspace` and `--fix` flag are implicit now) is not much longer than invoking a bash script.